lib/util/git/private-key.spec.ts

```diff
@@ -10,39 +12,121 @@ jest.mock('fs-extra', () =>
     >('../../../test/fixtures')
     .fsExtra(),
 );
-jest.mock('../exec');
+jest.mock('../exec', () =>
+  jest
+    .requireActual<typeof import('jest-mock-extended')>('jest-mock-extended')
+    .mock<MockProxy<typeof execModule>>(),
```

Comment:
```suggestion
    .mock(),
```
I think the type here doesn't matter?

why not use `mockDeep` as we use in other tests?

Reply:
> I think the type here doesn't matter?

You're right. I've removed the type: 0beb5e0c240520cfc715fde367768b445f328062

> why not use `mockDeep` as we use in other tests?

No specific reason, it just wasn't necessary. For better consistency with other tests, I've migrated to `mockDeep`: f7fa05de65e3e859e5092a3267779b7b949bd8f9